src/main/java/org/apache/mesos/hdfs/util/HDFSConstants.java

```diff
@@ -22,4 +22,8 @@
   public static final String NODE_EXECUTOR_ID = "NodeExecutor";
   public static final String NAME_NODE_EXECUTOR_ID = "NameNodeExecutor";
 
+  // Path to Store HDFS Binary
+  public static final String MESOSPHERE_OPT_PATH = "/opt/mesosphere/";
```

Comment:
Ok, sure makes sense. Ultimately, I would like this parameter to be configured or determined by Mesos or Mesosphere DCOS. It should be the default location for all framework binaries.